jekyll/_cci2/workflows.adoc

```diff
@@ -362,16 +362,73 @@ workflows:
             - test2
 ----
 
-The environment variables are defined by setting the `context` key as shown to the default name `org-global`. The `test1` and `test2` jobs in this workflows example will use the same shared environment variables when initiated by a user who is part of the organization. By default, all projects in an organization have access to contexts set for that organization.
+The `test1` and `test2` jobs have access to environment variables stored in the `org-global` context if the pipeline meets the restrictions set for the context, for example:
+
+* Was the pipeline triggered by a user who xref:contexts#security-goup-restrictions[has access] (is in the relevant org/security group etc.)?
+* Does the xref:contexts#project-restrictions[project have access] to the context? By default all projects in an organization have access to contexts set for that organization, but restrictions on project access can be configured.
+* Does the pipeline meet the requirements of any xref:contexts#expression-restrictions[expression restrictions] set up for the context?
+
+== Use conditional logic in workflows
+
+You may use a `when` clause (the inverse clause `unless` is also supported) under a workflow declaration with a <<logic-statements>> to determine whether or not to run that workflow.
+
+The example configuration below uses a pipeline parameter, `run_integration_tests` to drive the `integration_tests` workflow.
+
+[,yaml]
+----
+version: 2.1
+
+parameters:
+  run_integration_tests:
+    type: boolean
+    default: false
+
+workflows:
+  integration_tests:
+    when: << pipeline.parameters.run_integration_tests >>
+    jobs:
+      - mytestjob
+
+jobs:
+----
+
+This example prevents the workflow `integration_tests` from running unless the `run_integration_tests` pipeline parameter is `true`. For example, when the pipeline is triggered with the following in the `POST` body:
+
+[,json]
+----
+{
+    "parameters": {
+        "run_integration_tests": true
+    }
+}
+----
+
+[#using-filters-in-your-workflows]
+== Using filters in your workflows
+
+The following sections provide examples for using filters in your workflows to manage job execution.
+
+You can filter workflows by branch, git tag, both, or neither. Workflow filters for branches and tags have the keys `only` and `ignore`:
```

Comment:
Technically you cannot filter workflows by both branch and tag, only one can be applied. So I would remove "both" in this sentence.